web/src/components/LocationHeader.tsx

```diff
@@ -15,7 +15,9 @@ import {
   SHELTER_ROUTE,
   SPRUNGBRETT_OFFER_ROUTE
 } from 'api-client'
+import { config } from 'translations'
 
+import searchIconMobile from '../assets/IconSearch.svg'
```

Comment:
the old icons (desktop) gonna be replaced in a separate ticket for header redesign